docker/create-pr.sh

```diff
@@ -29,6 +30,6 @@ for ENV in $(echo $ENVS | tr "," " "); do
     done
     git commit -m "Update image tags for ${ENV} to ${IMAGE_TAG}"
     git push --set-upstream origin "${BRANCH}"
-    gh pr create --title "Update image tags for ${ENV} (${IMAGE_TAG})" --base main --head "${BRANCH}-${ENV}" --fill
+    gh pr create --title "Update image tags for ${ENV} (${IMAGE_TAG})" --base main --head "${BRANCH}" --fill
```

Comment:
You might find the PR workflow for automated commits a little tricky. For the rest of GOV.UK we just commit and merge directly. No problem if you wanna experiment with this though.

Reply:
I think lets try it out first, it will probably be mainly me who will be running the process